packages/core/src/shared-wallets/transaction/TransactionDetails.tsx

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable sonarjs/cognitive-complexity */
+/* eslint-disable complexity */
+/* eslint-disable no-magic-numbers */
+import { Box, InfoBar, InfoComponent, SummaryExpander, TransactionSummary } from '@input-output-hk/lace-ui-toolkit';
+import React, { useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { ActivityStatus } from '@src/ui/components/Activity';
+import { Transaction, TransactionFee, TxSummary } from '@src/ui/components/ActivityDetail';
+import { CosignerItemProps, CosignersList } from './CosignersList';
+import styles from './TransactionDetails.module.scss';
+
+export interface TransactionDetailsProps {
+  addressToNameMap: Map<string, string>;
+  amountTransformer: (amount: string) => string;
+  coinSymbol: string;
+  cosigners: CosignerItemProps['list'];
+  fee?: string;
+  handleOpenExternalHashLink?: () => void;
+  hash?: string;
+  includedDate?: string;
+  includedTime?: string;
+  isPopupView?: boolean;
+  ownAddress: string;
+  ownAddresses: string[];
+  signPolicy: {
+    participants: number;
+    quorum: number;
+  };
+  status?: ActivityStatus;
+  txInitiator: string;
+  txSummary?: TxSummary[];
+}
+
+export const TransactionDetails = ({
+  hash,
+  status,
+  fee = '-',
+  amountTransformer,
+  txSummary = [],
+  coinSymbol,
+  ownAddresses,
+  addressToNameMap,
+  isPopupView,
+  handleOpenExternalHashLink,
+  cosigners,
+  signPolicy,
+  includedDate,
+  includedTime,
+  txInitiator,
+  ownAddress,
+}: TransactionDetailsProps): React.ReactElement => {
+  const { t } = useTranslation();
+  const [isSummaryOpened, setIsSummaryOpened] = useState(true);
+  const [isCosignersOpened, setIsCosignersOpened] = useState(true);
```

Comment:
I think the correct form is `open`.
```suggestion
  const [isSummaryOpen, setIsSummaryOpen] = useState(true);
  const [isCosignersOpen, setIsCosignersOpen] = useState(true);
```

Reply:
done here da5017fa09f52aed18610ba5bf40c57b173aec76